assets/src/dashboard/app/views/myStories/content/storyGridItem/components.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * Internal dependencies
+ */
+import { CardGridItem } from '../../../../../components';
+
+export const CustomCardGridItem = styled(CardGridItem)`
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: ${({ $posterHeight }) => `${$posterHeight}px auto`};
+`;
+CustomCardGridItem.propTypes = {
+  $posterHeight: PropTypes.number.isRequired,
+};
+
+// Keeps grid item position contained
+export const Container = styled.div`
+  position: relative;
+`;
+
+// Allows grid item contents to overlay in a set space
+export const CardWrapper = styled.div`
+  position: absolute;
+  height: 100%;
+  width: 100%;
+`;
+
+// Displays story poster if available for story grid item as background of card area
+export const Poster = styled.div`
+  height: 100%;
+  width: 100%;
+  object-fit: fill;
+  border-radius: ${({ theme }) => theme.borders.radius.medium};
+  background: ${({ theme }) => theme.colors.gradient.placeholder};
+`;
+
+// Gradient overlays the story poster
+export const Gradient = styled.div`
+  position: absolute;
+  bottom: 0;
+  height: 67%;
+  width: 100%;
+  border-radius: ${({ theme }) => theme.borders.radius.medium};
+  background: ${({ theme }) => theme.colors.gradient.posterOverlay};
+`;
+
+// Holds story content
+export const Scrim = styled.div`
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  padding: 8px;
+  border-radius: ${({ theme }) => theme.borders.radius.medium};
+  background: ${({ theme }) => theme.colors.opacity.black3};
+`;
+
+// Components that create the content to display on top of a poster
+// Set guidance for how content should display on top of a poster
+export const StyledStoryDisplayContent = styled.div`
+  padding: 12px 4px 0 4px;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+`;
+
+// Generic row to vertically center grid item contents
+export const Row = styled.div`
+  width: 100%;
+  display: flex;
+  align-items: flex-end;
+  flex-wrap: wrap;
+`;
+export const LockedRow = styled(Row)`
+  position: absolute;
+  top: 8px;
+  right: 8px;
+  justify-content: flex-end;
+  align-items: center;
+`;
+
+// Interior grid for details to allow context menu space without jostling copy
+export const CardDetailsGrid = styled.div`
+  display: grid;
+  grid-template-columns: 80% auto;
+  margin-top: 4px;
+`;
+export const CardDetailsColumn = styled.div`
+  display: flex;
+  align-items: space-between;
+  flex-direction: column;
+  align-self: flex-end;
+`;
+
+// Story Title
+// Overriding stylelint for the sake of line clamping the title to be 3 lines.
+// This is supported by all major browsers except IE 11.
+// If it fails, max-height will catch so that really long titles
+// don't leave their designated space, just no ellipsis.
+export const Title = styled(Headline).attrs(() => ({
+  as: 'a',
+  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL,
+}))`
```

Comment:
Nit: I think this can just be an object since there are no props that it needs, right?
```suggestion
export const Title = styled(Headline).attrs({
  as: 'a',
  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL,
})`
```